app/subscriber.py

```diff
@@ -58,7 +58,28 @@ def receipt_to_case(message: Message):
     log = log.bind(questionnaire_id=questionnaire_id, created=time_obj_created, tx_id=tx_id, case_id=case_id)
 
     metadata['response_datetime'] = time_obj_created
-    send_message_to_rabbitmq(json.dumps(metadata))  # NB: in the future this should hold `questionnaire_id`
+
+    receiptMessage = {}
+    eventObject = {}
+    payloadObject = {}
+    responseObject = {}
+
+    eventObject['type'] = 'RESPONSE_RECEIVED'
+    eventObject['source'] = 'RECEIPT_SERVICE'
+    eventObject['channel'] = 'EQ'
+    eventObject['dateTime'] = time_obj_created
+    eventObject['transactionId'] = tx_id
+
+    responseObject['caseId'] = case_id
+    responseObject['questionnaireId'] = questionnaire_id
+    responseObject['unreceipt'] = 'false'
```

Comment:
Should be a `False` boolean object and let the json module serialise it?

Reply:
done